app/models/issue_importer.rb

```diff
@@ -81,6 +92,9 @@ def github_issue
   def import
     issue = Issue.by_owner_repo_number(owner, repository, number).first_or_initialize
 
+    # Return immediately if Issue has not changed since it was last updated.
+    return issue if github_issue.blank? && github_client.last_response.status == 304
```

Comment:
I use `github_issue.blank?` here since the response body when the resource has not been modified is `""`.